docs/api/amqp.md

```diff
@@ -172,6 +173,44 @@ JSON in the following format:
 }
 ```
 
+### Schema status
+
+Message with the status of schema update command.
+
+#### Exchange
+
+* `fog`
+
+#### Binding Key
+
+* `schema.updated`
+
+#### Expected Response
+
+JSON in the following format:
+  * `id` **String** device's ID
+  * `updated` **Boolean** updated result.
+  * `error` **String** (optional) a string with detailed error message.
+
+#### Success Response Example
+
+```json
+{
+  "id": "3aa21010cda96fe9",
+  "updated": true
+}
+```
+
+#### Error Response Example
+
+```json
+{
+  "id": "3aa21010cda96fe9",
+  "updated": false,
+  "error": ""
```

Comment:
Please, could you show some small traceback error example in the `error` field?

Reply:
Done